doc/entropy-seems-wrong.md

```diff
@@ -0,0 +1,107 @@
+Entropy Score Seems Wrong
+=========================
+
+When creating a password, your goal is to make it very difficult to guess or hack.  Sometimes you think your password is super secure because it is long.  Other times the password was randomly generated.  Maybe you are here because the entropy value seemed too high when you typed in "aaaaaaaa" or you were wondering why that's not the same entropy as "zzzzzzzz".
+
+Regardless the reason, let's dive a little into the math that is surrounding the entropy scores.
+
+
+Calculation of Entropy
+----------------------
+
+Researchers can spend their entire lives devising schemes to measure the amount of entropy in a given data set.  In essence, one only measures how much entropy there *isn't* instead of how much there *is*.
+
+
+### Entropy Basics
+
+For our examples, we will use 8 character long passwords and primarily restrict ourselves to the 26 lowercase letters in the English language.
+
+Assuming that you roll dice or have another truly random method for picking letters, you will generate one of a possible 208,827,064,576 passwords.  That number is generated because you have 26 different letters you can pick for each of the 8 letters in the password.  The math to compute the number of permutations would be 26^8, or 208 billion.
+
+This includes passwords like "thimbles" and "stupider", but more on that weakness later.
+
+To compute the number of bits of entropy, one just tries to figure out how many possibilities there are (208 billion) and then figures out how big of a binary number it would take to represent it.  To do that you use `log(26^8) / log(2)` becase normally the `log()` function provides results in base 10 and we require them in base 2.  This is also known as `log2(26^8)`, and the result is about 37.6 bits of entropy.
```

Comment:
You could use "To do this..." instead of "To do that..." Since you are talking about the same thing. Might need a comma after "To do this..."

"becase" is misspelled.